ScoreSaber/.editorconfig

```diff
@@ -0,0 +1,235 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
+root = true
```

Comment:
We should likely put the `.editorconfig` in the correct place rather than duplicating it.

Reply:
Had to reimport it for it to apply to my VS22, one of the .editorconfig could just be deleted. Or we can add the file to .gitignore instead?